src/components/organisms/NewProfileModal/components/buttons/ProfileModalButtons/ProfileModalButtons.tsx

```diff
@@ -28,7 +26,7 @@ export const ProfileModalButtons: React.FC<ProfileModalButtonsProps> = ({
     <ButtonNG
       // @debt until temporarily disable is online functionality
       variant={!isCurrentUser ? "primary" : "secondary"}
-      className={classNames("ProfileModalButtons", containerClassName, {
+      className={classNames("ProfileModalButtons", {
```

Comment:
```suggestion
      className="ProfileModalButtons"
```

Reply:
Had that thought. Decided to keep it because of the following code

```tsx
        // @debt until temporarily disable is online functionality
        // "ProfileModalButtons--online": !isCurrentUser && isOnline,
```